full-moon/src/ast/parsers.rs

```diff
@@ -349,6 +349,8 @@ define_parser!(
     ParseStmt,
     Stmt<'a>,
     |_, state: ParserState<'a>| parse_first_of!(state, {
+        @#[cfg(feature = "roblox")]
+        ParseCompoundAssignment => Stmt::CompoundAssignment,

```

Comment:
Same here--put this before `ParseTypeDeclaration` so that all the Roblox parsers are close together.